src/main/java/gregtech/common/pipelike/optical/net/OpticalPipeNet.java

```diff
@@ -38,13 +37,12 @@ public List<OpticalInventory> getNetData(BlockPos pipePos, EnumFacing facing) {
                 return Collections.emptyList();
             }
 
-            // TODO choose valid state by number of connected inventories - count does not include source tile
-//            if (data.size() == 1) {
-                data.sort(Comparator.comparingInt(inv -> inv.properties.hashCode()));
+            // only allow one other connection
+            if (data.size() == 1) {
```

Comment:
Might be better to return the walker early for potentially large nets, but not needed